rest/route/host.go

```diff
@@ -444,9 +445,31 @@ func (ch *offboardUserHandler) Run(ctx context.Context) gimlet.Responder {
 	if catcher.HasErrors() {
 		return gimlet.NewJSONInternalErrorResponse(errors.Wrapf(catcher.Resolve(), "not all unexpirable hosts/volumes terminated"))
 	}
+
+	if err := ch.clearLogin(); err != nil {
```

Comment:
This should consider dryRun, i.e. not clearLogin if dryRun is set. Also, should this not return an error to the user if it doesn't work? And what's the purpose of clearing this? Looking at implementations of (Manager) ClearUser I'm not seeing any functions that are actually implemented.

Reply:
Changed it to no-op for dryRun.

`ClearUserToken` is implemented by the [Okta auth manager](https://github.com/evergreen-ci/evergreen/blob/9f5e046c173b42fc27a12b609c7a296bae017b55/auth/okta.go#L41).

I don't think it's really that important to clear the login cache, but since there's no real harm in returning the error, I added it to the catcher.

Reply:
If it's not important to clear the login cache then it might not be worth returning the error since the only error we return right now are for important errors that Mana should hopefully retry for (i.e. unexpirable hosts/volumes not terminated)